cesium/build_model.py

```diff
@@ -148,6 +148,10 @@ def build_model_from_featureset(featureset, model=None, model_type=None,
         if inf_feats.any():
             message += " Inf in {}.".format(", ".join(feature_df.columns[inf_feats]))
         raise ValueError(message)
+    try:
+        model = model.best_estimator_
```

Comment:
@bnaul For one, it doesn't pickle/unpickle the same - using our existing code, if we let just return the `GridSearchCV` object here, `joblib.load(path)` errors. Also, calling `model.score()` on the `GridSearchCV` object errors.